bases/br_anp_precos_combustiveis/microdados/table_config.yaml

```diff
@@ -1,201 +1,346 @@
-        is_partition: False # Bool [True, False], whether the column is a partition.
-  
-    -   
-        name: ano
-        description: Ano
-        is_in_staging: True # Bool [True, False], whether the column is in the staging table
-        is_partition: True # Bool [True, False], whether the column is a partition.
+columns:
+    - name: ano
+      bigquery_type: int64
+      description: Ano
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id: br_bd_diretorios_data_tempo
+          table_id: ano
+          column_name: ano
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: true
+    - name: sigla_uf
+      bigquery_type: string
+      description: Sigla da Unidade Federativa da revenda pesquisada
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: uf
+          column_name: sigla
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_municipio
+      bigquery_type: string
+      description: ID Município IBGE - 7 Dígitos
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: municipio
+          column_name: id_municipio
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: bairro_revenda
+      bigquery_type: string
+      description: Nome do bairro da revenda pesquisada
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: setor_censitario
+          column_name: nome_bairro
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: cep_revenda
+      bigquery_type: string
+      description: Número do Código do Endereço Postal (CEP) do logradouro da revenda
+          pesquisada
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: endereco_revenda
+      bigquery_type: string
+      description: Endereço de revenda
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: cnpj_revenda
```

Comment:
Acho que vale verificar se é o CNPJ completo pra nomenclatura ficar igual a base de cadastro de cnpjs da receita federal. 
Se for só os 8 primeiros dígitos seria `cnpj_basico`
Se for o cnpj completo seria `cnpj`

Reply:
Eu conferi com a nossa base de cadastro de cnpjs, mas o cnpj da anp é o completo. Dessa forma, preferi manter o nome.